include/openthread/srp_server.h

```diff
@@ -194,6 +207,52 @@ otError otSrpServerSetDomain(otInstance *aInstance, const char *aDomain);
  */
 otSrpServerState otSrpServerGetState(otInstance *aInstance);
 
+/**
+ * This function returns the address mode being used by the SRP server.
+ *
+ * @param[in] aInstance  A pointer to an OpenThread instance.
+ *
+ * @returns The SRP server's address mode.
+ *
+ */
+otSrpServerAddressMode otSrpServerGetAddressMode(otInstance *aInstance);
+
+/**
+ * This function sets the address mode to be used by the SRP server.
+ *
+ * @param[in] aInstance  A pointer to an OpenThread instance.
+ * @param[in] aMode      The address mode to use.
+ *
+ * @retval OT_ERROR_NONE           Successfully set the address mode.
+ * @retval OT_ERROR_INVALID_STATE  The SRP server is enabled and the address mode cannot be changed.
+ *
+ */
+otError otSrpServerSetAddressMode(otInstance *aInstance, otSrpServerAddressMode aMode);
+
+/**
+ * This function returns the sequence number used with anycast address mode.
+ *
+ * The sequence number is included in "DNS/SRP Service Anycast Address" entry published in the Network Data.
+ *
+ * @param[in] aInstance  A pointer to an OpenThread instance.
+ *
+ * @returns The anycast sequence number.
```

Comment:
What's the return value if the SRP server is in mode `OT_SRP_SREVER_ADDRESS_MODE_UNICAST`?

Reply:
See above.